src/WaCore.Sample/Controllers/BooksController.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace WaCore.Sample.Controllers
 {
+#region UseUoWDocu
```

Comment:
Now the region contains the opening bracket of the class but not the closing bracket. I would start the region just before `private IUnitOfWork _unitOfWork;`

Reply:
That was intentional, otherwise it would be confusing that ``public BooksController(IUnitOfWork unitOfWork)`` is a constructor. I prefer to keep it like it is.

Reply:
I have a brilliant idea 😆
Make it a partial class and close it before the end of the region and define the rest of it outside the region.
Just a joke. Don't do that :-)